recipes/firefox/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+APP=${PREFIX}/bin/firefox
+
+if [[ $(uname) == Linux ]]; then
+  mkdir -p ${PREFIX}/bin/Firefox
+  mv * ${PREFIX}/bin/Firefox
+  cat <<EOF >$APP
+#!/bin/bash
+$PREFIX/bin/Firefox/firefox "\$@"
```

Comment:
This does not work. The PREFIX is not changed and we end up with `/opt/anaconda1anaconda2anaconda3/bin/Firefox/firefox`. Reading conda-build docs to see what we can do. (Ping @msarahan for help too ;-p)

Reply:
I don't understand your concern @ocefpaf, I've tested this locally and it works fine.

I end up with 

```
#!/bin/bash
/home/bird/miniconda3/envs/testff/bin/Firefox/firefox "$@"
```

I my environments /bin/firefox file, which is correct.

Reply:
It did not work for me locally :-/